pkg/log/checkpoint/queried.go

```diff
@@ -0,0 +1,100 @@
+package checkpoint
+
+import (
+	"errors"
+	"fmt"
+)
+
+const osSqlQuery = `
+SELECT
+	os_version.build as os_build,
+	os_version.name as os_name,
+	os_version.platform as os_platform,
+	os_version.platform_like as os_platform_like,
+	os_version.version as os_version
+FROM
+	os_version
+`
+
+const systemSqlQuery = `
+SELECT
+	system_info.hardware_model,
+	system_info.hardware_serial,
+	system_info.hardware_vendor,
+	system_info.hostname,
+	system_info.uuid as hardware_uuid
+FROM
+	system_info
+`
+
+const osquerySqlQuery = `
+SELECT
+	osquery_info.version as osquery_version,
+	osquery_info.instance_id as osquery_instance_id
+FROM
+    osquery_info
+`
+
+func (c *checkPointer) logOsqueryInfo() {
```

Comment:
should we add some data from `kolide_launcher_osquery_instance_history`? could possibly help up identify if osquery is crashing often and why

Reply:
That's a good thought. I'm going to merge that, and flag that for the next pass

Reply:
https://github.com/kolide/launcher/issues/1071